### PR TITLE
Fix AWS Security Hub parser to process AWS Inspector findings

### PR DESCRIPTION
**Pull request by manuel-sommer**
This is an enhancement for the AWS Security Hub parser to process AWS Inspector findings.

### CODE REVIEW

**Comment by manuel-sommer** [comment]
Thank you @quirinziessler for contributing. 
@Maffooch could you please review this PR? Could you rerun the Unit tests because of the failed one?

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by blakeaowens — APPROVED** [review]

File: dojo/tools/awssecurityhub/parser.py [hunk 1]
```diff
@@ -20,70 +21,87 @@ def get_findings(self, filehandle, test):
     def get_items(self, tree, test):
         items = {}
         # DefectDojo/django-DefectDojo/issues/2780
-        findings = tree.get("Findings", tree.get("findings", None))
+        findings = tree.get('Findings', tree.get('findings', None))
```
Comment (cneill) [inline dojo/tools/awssecurityhub/parser.py:24]: Please use double quotes consistently

File: dojo/tools/awssecurityhub/parser.py [hunk 2]
```diff
@@ -20,70 +21,87 @@ def get_findings(self, filehandle, test):
...
-    mitigation = (
-        finding.get("Remediation", {})
-        .get("Recommendation", {})
-        .get("Text", "")
-    )
-    references = (
-        finding.get("Remediation", {}).get("Recommendation", {}).get("Url")
-    )
-    false_p = False
+    aws_scanner_type = finding.get('ProductFields', {}).get('aws/securityhub/ProductName', "")
```
Comment (cneill) [inline dojo/tools/awssecurityhub/parser.py:38]: Please use double quotes consistently

**Review by cneill — COMMENTED** [review]
Please use double quotes consistently here. Thanks

File: dojo/tools/awssecurityhub/parser.py (hunk 2 above)
Comment (quirinziessler) [reply dojo/tools/awssecurityhub/parser.py]: thanks for your input. I just changed it with latest commit.

**Comment by manuel-sommer** [comment]
we need one more approving review. Could you take a look @grendel513 ?

**Review by cneill — APPROVED** [review]

**Pull request merged**